test/xref2/refs/refs.md

```diff
@@ -998,7 +998,7 @@ Reference to 'Y' is ambiguous. Please specify its kind: constructor-Y, module-Y.
    Y)
 # resolve_ref "Everything_ambiguous_in_sig.t" (* Some kinds are missing: label, type subst (would be "type-") *) ;;
 File "<test>":
-Reference to 't' is ambiguous. Please specify its kind: field-t, module-type-t, type-t, val-t.
+Reference to 't' is ambiguous. Please specify its kind: field-t, module-type-t, type-t, val-t, val-t.
```

Comment:
The two `val-` actually correspond to a `external` and a `val`. This is a bug in the `Find` module but instead of fixing it, I'm investigating removing that module entirely and using `Env` instead.

Reply:
Might be better to leave that change to 2.1.0?